packages/dd-trace/src/appsec/blocking.js

```diff
@@ -8,6 +8,9 @@ let templateLoaded = false
 let templateHtml = ''
 let templateJson = ''
 
+const ERROR_HTML = '<html>blocked</html>'
+const ERROR_JSON = '{"error": "blocked"}'
```

Comment:
You could just set these to the `template*` variables directly and skip needing these extra vars.

Reply:
Change has been made